src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/CreateLicenceResponse.kt

```diff
@@ -0,0 +1,17 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model
+
+import io.swagger.v3.oas.annotations.media.Schema
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceStatus
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceType
+
+@Schema(description = "Response object for a created licence")
+data class CreateLicenceResponse(
```

Comment:
I like to create subdirectories (request + response) inside the model/dto directory to split my DTOs into Request and Response DTOs, but personal preference only

Reply:
We can reorganise as we build up numbers of req/response objects? Next PRs?

Reply:
Yes no problem